### ISSUE TITLE
Document all MRview keyboard shortcuts

### ISSUE THREAD

**Issue by draffelt** [MRtrix3/mrtrix3#308]

**Comment by Lestropie** [comment]
Was just about to have a look into this, thinking the script for grabbing all of the config file parameters and creating `mrtrix_config_options.rst` would be the starting point; but it doesn't look like that script is in the repo, or didn't get ported in the transition from wiki to readthedocs. Anyone got it lying around?

**Comment by rtabbara** [comment]
Sorry @Lestropie , I think I added a script to generate .rst for the commands/scripts but not for the config params. For the initial port, I just did a manual copy from the wiki and hand-balled the automation to future me.

**Comment by Lestropie** [comment]
No probs; at least it's now listed with the milestone in tow.

Just was thinking in the context of #539, the config file options and `mrtrix` command-line options listings could go with the script and command listings in something like a 'reference' section.

**Comment by Lestropie** [comment]
Actually, assuming that "MRview shortcuts" is supposed to refer to the command-line shortcuts for configuring tools, this is no longer required: `mrview` moved away from the `-run ""` syntax to have a command line option group for each tool, and therefore these all appear in the `mrview` [help page](http://mrtrix.readthedocs.io/en/latest/getting_started/commands_list.html#mrview)...? Or was this supposed to refer to keyboard shortcuts (which are all visible in the drop-down menus)?

**Comment by jdtournier** [comment]
I think this was supposed to be about shortcuts... I agree that they're all easily discoverable via the menu, but it might also be nice to have a page on the documentation (maybe sandwiched between 'Images and other data' and 'command-line usage') with a brief intro/showcase of MRView's features, and how to access the help page to get the command-line options, and how to find out about shortcuts (it's amazing how often I have to point out to people that the shortcuts are shown in the menu or via tooltips...). This would be a bit like [what I had for the previous version](http://jdtournier.github.io/mrtrix-0.2/general/mrview.html). Obviously, that would take a bit of effort to write up though...

**Comment by Lestropie** [comment]
OK, in that case we'll leave this open. Though how crucial is it that this be done for release?

**Comment by jdtournier** [comment]
Agreed. I've just amended the title and remove the milestone. It can wait...
